File: samples/wordnik-api-java/com/wordnik/client/model/Definition.java
package com.wordnik.client.model;

public class Definition {
  private String text = null;
  private String source = null;
  private String note = null;
  private String partOfSpeech = null;
  private PartOfSpeech partOfSpeechObj = null;
  public String getText() {
    return text;
  }
  public void setText(String text) {
    this.text = text;
  }

  public String getSource() {
    return source;
  }
  public void setSource(String source) {
    this.source = source;
  }

  public String getNote() {
    return note;
  }
  public void setNote(String note) {
    this.note = note;
  }

  public String getPartOfSpeech() {
    return partOfSpeech;
  }
  public void setPartOfSpeech(String partOfSpeech) {
    this.partOfSpeech = partOfSpeech;
  }

  public PartOfSpeech getPartOfSpeechObj() {
    return partOfSpeechObj;
  }
  public void setPartOfSpeechObj(PartOfSpeech partOfSpeechObj) {
    this.partOfSpeechObj = partOfSpeechObj;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Definition {\n");
    sb.append("  text: ").append(text).append("\n");
    sb.append("  source: ").append(source).append("\n");
    sb.append("  note: ").append(note).append("\n");
    sb.append("  partOfSpeech: ").append(partOfSpeech).append("\n");
    sb.append("  partOfSpeechObj: ").append(partOfSpeechObj).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/Definition.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class Definition {
  @BeanProperty var text: String = _
  @BeanProperty var source: String = _
  @BeanProperty var note: String = _
  @BeanProperty var partOfSpeech: String = _
  @BeanProperty var partOfSpeechObj: PartOfSpeech = _
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class Definition {\n")
    sb.append("  text: ").append(text).append("\n")
    sb.append("  source: ").append(source).append("\n")
    sb.append("  note: ").append(note).append("\n")
    sb.append("  partOfSpeech: ").append(partOfSpeech).append("\n")
    sb.append("  partOfSpeechObj: ").append(partOfSpeechObj).append("\n")
    sb.append("}\n")
    sb.toString
  }
}
